packages/databases-collections-list/src/use-refresh.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button, Icon } from '@mongodb-js/compass-components';
+
+export type ItemType = 'database' | 'collection';
```

Comment:
Is this a leftover?

Reply:
yes that's right, thanks for pointing out